RELEASES.md

```diff
@@ -39,6 +39,8 @@ The plugin version is unchanged at `38` and is compatible with versions `v1.12.0
   - `avm.GetTxFee`
   - `platform.getValidatorFeeConfig`
   - `platform.getValidatorFeeState`
+  - `validationID` to `platform.getL1Validator` results
+  - L1 validators to `platform.getCurrentValidators`
```

Comment:
nit just a tiny bit clearer imo:

```suggestion
  - L1 validators are now listed in `platform.getCurrentValidators`
```

Reply:
I think it contradicts with `-Added: ` above